### PR TITLE
Fix issue displaying multiple tickets (erroneous </div>) and also sim…

### PR DESCRIPTION
**Pull request by abobwhite**
…plified logic/fixed linting things.

Changed local dev to 8000 since 9000 was conflicting with something. Added simpler npm scripts for development (try not to use grunt globally)

### CODE REVIEW

File: app/scripts/controllers/citationInfo.ctrl.js [hunk 1]
```diff
@@ -1,192 +1,196 @@
...
-
+  ctrl.issueMultiplePeopleWarning = function () {
     return (_.size(ctrl.groupedCitations) > 1);
   };
 
-  ctrl.selectCitation = function(citation,idToScrollTo){
+  ctrl.selectCitation = function (citation, idToScrollTo) {
     ctrl.selectedCitation = citation;
     Session.setSelectedCitation(ctrl.selectedCitation);
-    if (ctrl.selectedCitation) {
```
Comment (ctwoolsey) [inline app/scripts/controllers/citationInfo.ctrl.js]: Why did you remove ctrl.selectedCitation check?

File: app/scripts/controllers/ticketFinder.ctrl.js [hunk 2]
```diff
@@ -61,14 +61,6 @@ angular.module('yourStlCourts').controller('TicketFinderCtrl', function (TicketF
     }
   };
 
-  function numberOfMunicipalitiesSelectedIsValid(){
```
Comment (ctwoolsey) [inline app/scripts/controllers/ticketFinder.ctrl.js]: Why did you remove this function?

Comment (abobwhite) [reply app/scripts/controllers/ticketFinder.ctrl.js]: This function was not used anywhere. The IDE showed me that easily

File: app/scripts/controllers/citationInfo.ctrl.js (hunk 1 above)
Comment (abobwhite) [reply app/scripts/controllers/citationInfo.ctrl.js]: If you'll see the new initialize logic, I no longer allow the selected citation to be null/undefined so the check was no longer necessary

File: test/spec/controllers/citationInfo.ctrl.spec.js [hunk 3]
```diff
@@ -221,12 +221,6 @@ describe('CitationInfoCtrl', function () {
     expect (CitationInfoCtrl.showPaymentButton()).toBe(false);
   }));
 
-  it('correctly sets selected citation when citation is null', inject(function () {
-    CitationInfoCtrl.selectCitation(null);
-
-    expect(CitationInfoCtrl.selectedCitation).toBe(null);
-  }));
```
[the remark this answers is not in the document]
Comment (abobwhite) [reply test/spec/controllers/citationInfo.ctrl.spec.js]: See above reply. And see newly added test in this same file
